test/Microsoft.ML.Tests/Scenarios/Api/CookbookSamples/CookbookSamples.cs

```diff
@@ -42,7 +42,7 @@ private void IntermediateData(string dataPath)
             var mlContext = new MLContext();
 
             // Create the reader: define the data columns and where to find them in the text file.
-            var reader = TextLoader.CreateReader(mlContext, ctx => (
+            var reader = mlContext.Data.Reader(ctx => (
```

Comment:
>mlContext.Data.Reader [](start = 25, length = 21)

When you change this, please also change the corresponding lines in `docs/code/MlNetCookBook.md`